app/views/comfy/admin/api_forms/_form.html.haml

```diff
@@ -56,8 +56,8 @@
                     = a.select :field_type, options_for_select(['input', 'textarea'], properties[key]["field_type"] || 'input'), {}, {class: 'form-control' }
                   .form-group.col-12
                     = a.label :type_validation, 'Type validation', class: 'mr-4'
-                    = a.select :type_validation, options_for_select(ApiForm::INPUT_TYPE, properties[key]["type_validation"] || 'text'), {}, {class: 'form-control', onchange: "toggleRegex(this , '#{key}_regex')" }
-                  .form-group.col-12{style: "#{'display: none' unless properties[key]["type_validation"] == 'REGEX pattern' }", id: "#{key}_regex"}
+                    = a.select :type_validation, options_for_select(ApiForm::INPUT_TYPE_MAPPING.keys, properties[key]["type_validation"] || 'text'), {}, {class: 'form-control', onchange: "toggleRegex(this , '#{key}_regex')" }
+                  .form-group.col-12{style: "#{'display: none' unless properties[key]["type_validation"] == 'regex_pattern' }", id: "#{key}_regex"}
```

Comment:
you should use the constant you defined [here](https://github.com/restarone/violet_rails/pull/266/files#diff-115eaea86e7fb2e74e1045aa541f956134ae663024878679d1a3e82731f53fd7) instead of hard coding `regex_pattern`

Reply:
Made the change, Thanks